h2o-core/src/main/java/water/rapids/ast/prims/math/AstUniOp.java

```diff
@@ -31,6 +31,12 @@ public Val apply(Env env, Env.StackHelp stk, AstRoot asts[]) {
         return new ValNum(op(val.getNum()));
       case Val.FRM:
         Frame fr = val.getFrame();
+        //Get length of columns in fr and append `op(colName)`. For example, a column named "income" that had
+        //a log transformation would now be changed to `log(income)`.
+        String[] newNames = new String[fr.names().length];
+        for(int i = 0; i < newNames.length; i++){
+          newNames[i] = asts[0].str() + "(" + fr.name(i) + ")";
```

Comment:
It's dangerous to use asts in their raw form (i.e. without evaluation / tracking). In this particular case `asts[0]` was already evaluated, and its value is `this`. So you could write simply `str()`.